lib/project_types/rails/messages/messages.rb

```diff
@@ -96,12 +96,12 @@ module Messages
             deployed: "{{v}} Deployed to Heroku",
             db_check: {
               validating: "Validating application...",
-              checking: "Checking database type...",
-              validated: "Database type \"%s\" validated for platform \"Heroku\"",
-              problem: "A problem was encountered while checking your database type.",
+              checking: "Checking database system...",
```

Comment:
Rails refers to this as "database system", I think we should follow the same naming in case people google it.